src/chi/components/badge/badge.scss

```diff
@@ -81,13 +81,14 @@ $colors: (
 .chi-badge {
   background-color: $color-background-white;
   border: 0.0625rem solid $color-border-base-dark;
-  border-radius: $border-radius;
+  border-radius: 0.75rem;
```

Comment:
there's a better way to do this using ems instead of rems but it will require more testing and changes to the badge component styles. i'll create a separate ticket/PR for that and we can include it in the following release. it will also enable support for text utility classes on badges.